eng/intellisense.targets

```
@@ -0,0 +1,41 @@
+<Project>
+
+  <PropertyGroup>
+    <XmlDocFileRoot>$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)', 'microsoft.private.intellisense', '$(MicrosoftPrivateIntellisenseVersion)', 'IntellisenseFiles'))</XmlDocFileRoot>
+  </PropertyGroup>
+
+  <!-- If the project is not generating its own documentation yet, retrieve it from the dotnet-api-docs nuget package. -->
+  <Choose>
+    <When Condition="('$(UseIntellisenseDocumentationFile)' == '' or '$(UseIntellisenseDocumentationFile)' == 'false') and Exists($([MSBuild]::NormalizePath('$(XmlDocFileRoot)', 'net', '1033', '$(AssemblyName).xml')))">
```

MSBuild style nit: Properties that are used for boolean feature flags should only contain two values: empty (which means false) or 'true'.

```suggestion
    <When Condition="'$(UseIntellisenseDocumentationFile)' != 'true' and Exists($([MSBuild]::NormalizePath('$(XmlDocFileRoot)', 'net', '1033', '$(AssemblyName).xml')))">
```